### PR TITLE
Men fashion style page

### PR DESCRIPTION
**Pull request by Salam-Dalloul**
men fashion page style done!

#36

### CODE REVIEW

File: public/css/menFashion.css [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+display: flex;
+justify-content: flex-end;
+}
+.headerForm{
+display: flex;
+align-items: center;
+align: left;
+margin-right: 10px;
+}
+input{
```
Comment (walaamedhat) [inline public/css/menFashion.css]: I think you should give this a class name and every thing you make a style for it like as footer,header

File: public/css/menFashion.css [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+padding-right: 15px;
+height: 70px;
+background-color: #FAFAFA;
+align-items: center;
+justify-content: center;
+}
+
+.navigationItems{
+  justify-content: center;
+  /*padding-right: 240px;*/
```
Comment (walaamedhat) [inline public/css/menFashion.css:64]: delete a comment from here :+1:

File: public/menFashion.html [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+        <li><a href="#">Contact us</a></li>
+      </ul>
+    </div>
+    <img src="./image/star.png" class="star" alt="">
+</div>
+</div>
+<div class="boxs container">
+<div class="box">
+  <div class="innerbox">
+    <a href="#"><img src="./image/men/men1.png" alt=""></a>
```
Comment (walaamedhat) [inline public/menFashion.html]: give every image alt description :+1:

**Review by walaamedhat — CHANGES_REQUESTED** [review]
please review changes

File: public/css/menFashion.css [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+height: 100vh;
+width: 100%;
+}
+
+.container{
+  width: 1140px;
+  flex-wrap: wrap;
+
+}
+.containernav{
```
Comment (MahmoudMH) [inline public/css/menFashion.css:13]: you should choose method naming classes camel case or other and use it in all naming  plz

File: public/css/menFashion.css [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+  justify-content: space-between;
+  flex-wrap: wrap;
+
+}
+.bigContainer{
+  display: flex;
+  flex-direction: column;
+    flex-wrap: wrap;
+}
+header{
```
Comment (MahmoudMH) [inline public/css/menFashion.css]: use classes and don't use in Css name of tag >> mush told me that . 
@Salam-Dalloul

File: public/menFashion.html [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+    <a href="#"><img src="./image/men/men7.png" alt=""></a>
+  </div>
+  <p>text</p>
+  <p class="info">120 NIS <i class="fa fa-heart-o"></i> </p>
+</div>
+<div class="box">
+  <div class="innerbox">
+    <a href="#"><img src="./image/men/men2.png" alt=""></a>
+  </div>
+  <p>text</p>
```
Comment (MahmoudMH) [inline public/menFashion.html]: we should to delete this text for each element on the page

File: src/database/build.js [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+  connection.query(sql, (errDb, result) => {
+    if (errDb) {
+      console.log(errDb, 'Error');
+    } else {
+      console.log('Database Created');
+    }
+  });
+};
+buildDatabase();
+module.exports = buildDatabase;
```
Comment (MahmoudMH) [inline src/database/build.js:16]: we are already editing this in the master

**Review by MahmoudMH — CHANGES_REQUESTED** [review]

**Review by walaamedhat — APPROVED** [review]

**Pull request closed without merging**